lambda_src/drivers/process_sentry_logger.py

```diff
@@ -113,5 +116,8 @@ def process_row(
             sentry_sdk.set_tag('error_time', ts)
             sentry_sdk.set_tag('history_type', history_type)
             SENTRY_DRIVER_LOGGER.exception(error)
+        status = f'Captured {name} error at {ts}.'
     except Exception as e:
```

Comment:
this is a little confusing -- which exceptions are you looking to capture here, exactly?

Reply:
exceptions using the `sentry_sdk`.

Reply:
moved a different PR.